tests/test_pipeline_media.py

```diff
@@ -31,7 +31,7 @@ def setUp(self):
 
     def tearDown(self):
         for name, signal in vars(signals).items():
-            if not name.startswith('_'):
+            if not name.startswith('_') and name is not 'Signal':
```

Comment:
Use `name != 'Signal'` here. The `is` operator does not check for equality, but for identity (i.e. "does this point to the very same position in memory?"). Using `is` when you mean `==` will sometimes work b/c CPython caches some string and number literals, but you can never be sure:

```python
In [1]: teststr = "hello"
In [2]: teststr is "hello"
Out[2]: True
In [3]: teststr += "2"
In [4]: teststr is "hello2"
Out[4]: False
In [5]: teststr == "hello2"
Out[5]: True
```